src/tests/routes/users.spec.ts

```diff
@@ -105,3 +105,30 @@ describe('[CREATE] POST: /users', () => {
   });
 });
 
+describe('[DELETE] DELETE: /users/:id', () => {
+  it('should return 404 status if user don\'t exists', async () => {
+    const newObjectId = mongoose.Types.ObjectId();
+    const result = await chai.request(app)
+        .delete(`/users/${newObjectId}`);
+    expect(result).to.have.property('status', 404);
+    expect(result).to.have.property('body');
+    expect(result.body).to.be.deep.equals({message: 'Utente non trovato'});
```

Comment:
Verificare con oggetto custom tipo 
```suggestion
    expect(result.body).to.be.deep.equals(expectedNotFoundResponse);
```